source/includes/fundamentals/code-snippets/CollationCollectionExample.java

```diff
@@ -183,18 +219,20 @@ private static void customCollationOperation(MongoCollection collection) {
     }
 
     private static void operationCollation(MongoCollection collection) {
+        // Runs a find operation that meets the criteria for using the "name" field index
```

Comment:
Suggestion:

I think it would be good to keep consistent with the other instances of find within this file and the [code comments sheet](https://docs.google.com/spreadsheets/d/1gCIeV5Bsp1G69MUNQZm8bYK9UNVSQjJt9T7eUM8aJiE/edit#gid=0):

```suggestion
        // Retrieves documents that match the query filter and applies a collation and ascending sort to the results
```